docs/content/templates/variables.md

```diff
@@ -158,6 +158,7 @@ Also available is `.Site` which has the following:
 **.Site.Permalinks** A string to override the default permalink format. Defined in the site configuration.<br>
 **.Site.BuildDrafts** A boolean (Default: false) to indicate whether to build drafts. Defined in the site configuration.<br>
 **.Site.Data**  Custom data, see [Data Files](/extras/datafiles/).<br>
+**.Site.DisableSearchJSON**  A boolean (by default false) to indicate wether to build a content index. Defined in the site configuration. Read more about the [search]({{< relref "templates/search.md" >}}) feature.<br>
```

Comment:
Sorry, old sed/perl syntax shorthand.  In english: wherever you have "wether", substitute "whether".